demos/README.md

```diff
@@ -12,5 +12,5 @@ The `demos` directory of this repo includes a collection of guided, step-by-step
 | [Simple Loan Marketplace](finserv-and-insurance/simple-loan-marketplace/README.md) | Basic | Highlights the ease of creating a multi-party data sharing solution, using the Vendia Share platform, as applied to the Financial Services industry
 | [Simple Food Supplier/Distributor Network](food-and-beverage/simple-supplier-distributor/README.md) | Basic | Highlights the ease of creating a multi-party data sharing solution, using the Vendia Share platform, as applied to the Food Services industry
 | [Alliance Ticket Sharing](travel-and-hospitality/alliance-ticket-sharing/README.md) | Intermediate | highlights the purpose and ease of creating data access controls for multi-party data sharing solutions, using the Vendia Share platform, as applied to the Travel and Hospitality industry
-| [CRM Data Sharing](crm-sharing/README.md) | Intermediate | Highlights the relative ease with which business partners can share account data from their respective Customer Relationship Management (CRM) systems in a controlled manner
+| [CRM Data Sharing](crm-sharing/) | Intermediate | Highlights the relative ease with which business partners can share account data from their respective Customer Relationship Management (CRM) systems in a controlled manner
```

Comment:
Can you tell me more about why the direct link to the README.md file isn't desired here?  It seems like we should either keep it as-is here or adjust all of the other links in the same way.

Reply:
Certainly. Because `crm-sharing/README.md` does not exist. This link is suppose to direct the reader to the CRM Data sharing folder. But clicking on it will route to a 404 page which is not desired.

We can also do:

1. Create a `README.md` file under `crm-sharing`. Though I wouldn't know what to put in it.
2. Break this into 2 and route them to `crm-sharing/console-experience/README.md` and `crm-sharing/postgres-to-salesforce/README.md`. 

Let me know what you think and I'm happy to change them. :)

Reply:
Option 2 is what we did for https://github.com/vendia/examples/tree/main/demos/food-and-beverage so let's do that for CRM as well.

Reply:
Complied with